.travis.yml

```diff
@@ -1,36 +1,39 @@
+- PR checks ✅
+- name: "Ship It \U0001F6A2"
+  if: branch=master AND type=push
 jobs:
   include:
-    - stage: PR checks ✅
-      env:
-        - NAME=Unit Tests
-      script: npm run test
-      after_script:
-        - codecov
-    - stage: PR checks ✅
-      env:
-        - NAME=Validation Checks
-      script: ./.travis/validate_changes_for_merge.sh
-    - stage: Ship It 🚢
-      env:
-        - NAME=Publish
-      script: semantic-release
-
+  - stage: PR checks ✅
+    env:
+    - NAME=Unit Tests
+    script: npm run test
+    after_script:
+    - codecov
+  - stage: PR checks ✅
+    env:
+    - NAME=Validation Checks
+    script: "./.travis/validate_changes_for_merge.sh"
+  - stage: "Ship It \U0001F6A2"
+    env:
+    - NAME=Publish
+    script: semantic-release
 notifications:
   webhooks: https://www.travisbuddy.com/
+  hipchat:
+    rooms:
+      secure: jLi64vC05Chtjofdqs2GwdsIoZJn2OE8Nk+TBqRGwAWkRQ6UVP9kbyxZYTB7y0Jr60x2grUs9m3sQycjZrAy0Me4ge+pfAuVhim98RiAVOUPeQGRJujXoIAR4xpB8FTMsmW3vGuAz5v5cq+jKm0i6c4/5Jqn7wPkGCVYDUApmriRKuMgKsbx8FSC4kYWsRBaDEOuFU2P8LiMHaJTVZVEhnSKlJelyi8CYK7if5U82A501kuYYFbDgd24BQunkFsKL881YZ7hBjnY/JzMIXxGfLtIyUq61SQQjpu1ccum9nw+8DoJ52jTFTulgfrLupi3InOKaRzjq1POnGiZyrXEedGCWvH4lb61fKlL9OYp5yExvPlClsdwLFq+3EZhwWkquWeCftvTECdjIX6k2wFCaFl9VnkF8RCPfQioVKdE2qk8mEUsQHIjEt/84q/qoJGl5pgw97sS/+JkYmB4eytrSFwP6H9uDrRRutAp2EMHPbmYUTvWeDEOwjaHLb1iChvkkxMDydY2bO1EHWrLXl0R4QT8lOub7WRKQywgtopNLJWc+FZtntOjb5YX7rLnrFSXH/G2nczSvWScNOzqNrXzS/y1SvDoFw2dm1gO8RXhYwwYPJPqLAmFMXEClAdScPehWn8qG9YV1KI/L1PTktYX4RxCUjSKbOMDZi4vP984fdo=
+env:
+  global:
+  - secure: pMK9vTOzgE+FdmpHUTJuVyVQagJxqD+hCueNjbOLlUeg7YhAoFX+Vf1pELXRTzWoeb8vCcakMkS1ovR5ZqzHm50u2UOpHBqGro38/lRegk/WBA4klrbgTwXZZYmREgfU2lx2mKEjJIynxvSpPMN3tv00z18JvBprUI/ngul+x2xrsAi8P/ro2PxXq2V1xjh0K/06FwYLzYQW3Cee88CpbuH3UKJvemw26FUxZFonc5QiVcHWuQuztIL2LWD5Miqnc5oGNTULFHSwS4yihK9WT/67tGkPJfO/BhgPfTL9LErG0bBU+kBRairYtSES4L8FprycGhT9usgGjGuaZrqrqtI06QywaI5OtPiN/QSqedt6oYRCUg44lC/RW0ZYMxG5K1Y8070+8WIZw/Xk6wLU7gHGovsTRR4xO5Dvj2myK4nBUgze7CeMD+//P638hdmqJ+8/QhUjdCphVBwJsdpOcLWm76GjUR6OZ+HcUHa42NyeDP8C7kbW2NcWWiqeTXMJ+jp9YYHeg/jdLeQPRMydZTZcjwIDwCzAFnLbEpd8aLPImKfFD4QokU24FrWtrLOg0YSP0GJ2gzhR9ZoVZfH4NEHF2BQ6R0/N52lfG7aXOUHFjcUBhkGgtTFTk71+fksLCFF7UplszybhUkUmyttIN0LYYG6ipPfWJA85hnY0eFM=
+  - secure: KA9Hgej0FmwlF7gbEmwUBLTaQi/PJa9RfDl9/cYAAhU/2ogQr6E9ZnDW0siWyl/CHauvw38bfx17+Ztf2+5ABDC8V9Id1stvKb+k4prM+s/z2MgzbfJMSgOU3oRja+pKv1FzQSkj+8zGfWuwTs8DSz4jxdW3/im0dd3RtCQBP+ztVn2pauzbYsY7+PjTMKYl7Ow0398id6L0+GTS+jJ+elF5OEP226n0SJJjqUmF2niLzNHNtUmnhk/Rkb0mjwHlzyUggsTlNvKWnwcF/fcEmHprUQ7ygjiRiY2nvlP5UM+kFuVtvYY9Zjmual7MDAW5mdguTw3MpbaY35oqnRYOSis1DhikskhvYogFHxR/0I0XJrnKvx6RJJuf0ME2RLeSQ3u8Oe0hjvCdjFdZTqiv8ma5XKnann/D+a2t3g8qkoFCr0nm4twZtLVECp8AdCioogBwoCPKmVjT4fNtkAOqoSfbCrUV9OfS0oOn+WTU+sb1oF/j7kOvhg/eR8ASXuoim7C9pKF7IuSS4BupkzExmUJi1zr9Ehi8UNkJt1859hkpQQL7BPnRvffXkD406s80DaKpzal5bQmX0ALiTf2ozFzrJnOxR/lF7oRYBlT1OuDQ66hG5j+jSTTYTVnHrzrBjMY09tlC546MA9VMrZEnp0tqqY8GpgOoaoKvWjdfUlc=
```

Comment:
Am I correct in assuming these values are `NPM_TOKEN` and `GH_TOKEN`?

I've deleted the set-on-travis-web-ui values, so we'll see how this works once it merges.

Reply:
yep